policyengine_us/variables/household/demographic/tax_unit/tax_unit_grandparents.py

```diff
@@ -0,0 +1,11 @@
+from policyengine_us.model_api import *
+
+
+class tax_unit_grandparents(Variable):
+    value_type = int
+    entity = TaxUnit
+    label = "Number of grandparents in the tax unit"
+    definition_period = YEAR
+
```

Comment:
can replace formula, same with parents
```suggestion
    adds = ["is_grandparent_of_filer_or_spouse"]
```